content/utils.js

```diff
@@ -1,12 +1,21 @@
 const path = require("path");
 const childProcess = require("child_process");
 
-const { CONTENT_ROOT } = require("./constants");
+const { CONTENT_ROOT, CONTENT_TRANSLATED_ROOT } = require("./constants");
 const { slugToFolder } = require("../libs/slug-utils");
 const LRU = require("lru-cache");
 
 const MEMOIZE_INVALIDATE = Symbol("force cache update");
 
+function getRoot(locale, throws = "") {
+  const root =
+    locale.toLowerCase() === "en-us" ? CONTENT_ROOT : CONTENT_TRANSLATED_ROOT;
+  if (throws && !root) {
+    throw new Error(throws);
+  }
```

Comment:
This is a bit weird. The only caller that ever uses this is the call from `redirects.js` and it could easily be refactored to:
```
const root = getRoot(locale)
if (!root) {
  throw new Error(`trying to add redirects for ${locale} but CONTENT_TRANSLATED_ROOT not set`)
}
```

Reply:
Matter of taste, 0 feelings. I think there will be more places where we wanna enforce `CONTENT_TRANSLATED_ROOT` to be set. But if you care, I'm happy to change this.